package.json

```diff
@@ -22,18 +22,19 @@
   },
   "homepage": "https://github.com/not-an-aardvark/eslint-plugin-prettier#readme",
   "dependencies": {
-    "prettier": "^0.11.0",
     "requireindex": "~1.1.0"
   },
   "peerDependencies": {
-    "eslint": "^3.14.1"
+    "eslint": "^3.14.1",
+    "prettier": ">= 0.11.0"
   },
   "devDependencies": {
     "eslint": "^3.14.1",
     "eslint-config-not-an-aardvark": "^2.0.0",
     "eslint-plugin-eslint-plugin": "^0.2.1",
     "eslint-plugin-node": "^3.0.5",
-    "mocha": "^3.1.2"
+    "mocha": "^3.1.2",
+    "prettier": ">= 0.11.0"
```

Comment:
I suppose I'll pin the version of `prettier` that this module uses to lint itself so that the build doesn't unexpectedly break.